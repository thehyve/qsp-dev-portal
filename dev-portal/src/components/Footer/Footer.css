.Footer {
  background-color: #5bc0de;
  color: #fff;
  position: fixed;
  right: 0;
  bottom: 0;
  left: 0;
  height: 55px;
  padding: 1rem;
  text-align: center;
  width: 100%;
}
