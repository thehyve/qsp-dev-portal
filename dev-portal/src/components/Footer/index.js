import React, {PureComponent} from 'react'
import './Footer.css'

export default class Footer extends PureComponent {

  constructor(props) {
    super(props);
  }

  render() {
    return (
      <footer className="Footer">
        <small>&copy; 2018 QUISPER</small>
      </footer>
    )
  }
}
