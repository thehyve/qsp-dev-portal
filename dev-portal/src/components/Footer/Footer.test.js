// Footer.test.js

import React from 'react';
import renderer from 'react-test-renderer';
import Footer from "./index";

test('should render Footer without any problem', () => {
  const testRenderer = renderer.create(
      <Footer />
  );
  expect(testRenderer).toMatchSnapshot();
});

