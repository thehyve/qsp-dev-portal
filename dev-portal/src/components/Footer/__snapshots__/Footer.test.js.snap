// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer without any problem 1`] = `
<footer
  className="Footer"
>
  <small>
    Â© 2018 QUISPER
  </small>
</footer>
`;
